currency_rate_update/models/res_currency_rate_provider.py

```diff
@@ -112,9 +112,6 @@ def _compute_available_currency_ids(self):
                 [("name", "in", provider._get_supported_currencies())]
             )
 
-    def _get_close_time(self):
```

Comment:
Why removing this?

Reply:
I mean, on weekends, there's no data, so are you sure this is going to work?

Reply:
Is the approach correct now?

Reply:
If you don't explain more what you want to achieve, I'm not sure.